lib/doorkeeper/grant_flow.rb

```diff
@@ -0,0 +1,54 @@
+require 'doorkeeper/grant_flow/flow'
+
+module Doorkeeper
+  module GrantFlow
+    mattr_accessor :registered_flows
+    self.registered_flows = {}
+
+    module_function
+
+    def register(name_or_flow, options = {})
+      if name_or_flow.is_a? Doorkeeper::GrantFlow::Flow
+        registered_flows[name_or_flow.name.to_sym] = name_or_flow
```

Comment:
In that case, let's add sample code as documentation in the README or in a code comment. Thank you! :+1: